frontend/src/metabase/components/Popover.jsx

```diff
@@ -72,96 +74,195 @@ export default class Popover extends Component {
     noOnClickOutsideWrapper: false,
   };
 
-  _getPopoverElement(isOpen) {
-    if (!this._popoverElement && isOpen) {
-      this._popoverElement = document.createElement("span");
-      this._popoverElement.className = "PopoverContainer";
-      document.body.appendChild(this._popoverElement);
-      this._timer = setInterval(() => {
-        const { width, height } = this._popoverElement.getBoundingClientRect();
-        if (this.state.width !== width || this.state.height !== height) {
-          this.setState({ width, height });
-        }
-      }, 100);
-    }
-    return this._popoverElement;
+  _createPopoverElement() {
+    const el = document.createElement("span");
```

Comment:
We could use this opportunity to stop using `span` for things that are not inline-elements and switch to `div`.

Reply:
Good thinking, done